android/src/main/java/io/mosip/tuvali/verifier/GattService.kt

```diff
@@ -47,38 +48,45 @@ class GattService {
       BluetoothGattCharacteristic.PERMISSION_WRITE
     )
 
-    val responseChar = BluetoothGattCharacteristic(
-      RESPONSE_CHAR_UUID,
+    val submitResponseChar = BluetoothGattCharacteristic(
+      SUBMIT_RESPONSE_CHAR_UUID,
       BluetoothGattCharacteristic.PROPERTY_WRITE_NO_RESPONSE or BluetoothGattCharacteristic.PROPERTY_WRITE,
```

Comment:
@PuBHARGAVI Are both types of write properties required here?

Reply:
@krishnakumar4a4 on ios side we are writing to "SUBMIT_RESPONSE_CHAR_UUID" characteristic with response `connectedPeripheral.writeValue(messageData, for: characteristic, type: .withResponse)` and android side we are writing to it without response `writeChar?.writeType = BluetoothGattCharacteristic.WRITE_TYPE_NO_RESPONSE`